src/Spectre.Console/Prompts/SelectionPrompt.cs

```diff
@@ -53,6 +58,11 @@ public sealed class SelectionPrompt<T> : IPrompt<T>, IListPromptStrategy<T>
     /// </summary>
     public SelectionMode Mode { get; set; } = SelectionMode.Leaf;
 
+    /// <summary>
+    /// Gets or sets a value indicating whether or not the search filter is enabled.
+    /// </summary>
+    public bool SearchFilterEnabled { get; set; } = false;
```

Comment:
As it's already the default:

```suggestion
    public bool SearchFilterEnabled { get; set; }
```